onnxruntime/contrib_ops/cuda/bert/relative_attn_bias_impl.cu

```diff
@@ -279,7 +279,7 @@ Status LaunchGatedRelativePositionBiasKernel(
         reinterpret_cast<vec_type*>(output),
         reinterpret_cast<const vec_type*>(rel_pos),
         qw, bias, eco_a, D, ldqw, equiv_seq_len);
-  } else if (seq_len & 1 == 0) {
+  } else if ((seq_len & 1) == 0) {
```

Comment:
This change fixed a bug.  If without the parentheses， “1 == 0” will be evaluated first.